content/03.results.md

```diff
@@ -224,46 +235,47 @@ The mutual exclusivity of signatures 3 and MMR corroborates a previous report su
-However, we also found that higher expression of _CD274_ is significantly associated with better overall prognosis (HR = 0.00074, CI = 3.0e-6 to 0.18, p = 0.01).
-Consistent with a previous study, we found higher expression of _CD274_ in the WNT subgroup, which has the best prognosis of all medulloblastoma subgroups [@doi:10.1080/2162402X.2018.1462430] (**Figure {@fig:Fig5}E**).
-Finally, we asked whether any molecular subtypes might show an immunologically hot phenotype, as roughly defined by a greater proportion of CD8+ to CD4+ T cells and while we do see the highest ratios in Group 3 and Group 4 medulloblastomas (**Figure {@fig:S6}F**), very few tumors had ratios > 1, highlighting an urgent need to identify novel therapeutics for these immunologically cold pediatric brain tumors with poor prognosis.
-
-![**Transcriptomic and immune landscape of pediatric brain tumors** A, First two dimensions from UMAP of sample transcriptome data. Points are colored by `broad_histology` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, Box plots of quanTIseq estimates of immune cell proportions in histologies with N > 15. D, Forest plot depicting the effect of immune cell proportion or _CD274_ expression on overall survival of medulloblastomas. Hazard ratio (HR) with 95% confidence intervals and p-values are listed and significant p-values are denoted with colored diamonds. Of note, the Macrophage M1 HR was 0 (coefficient = -8.95e04) with infinite upper and lower CIs, thus it was not plotted. E, Box plot of _CD274_ expression (log<sub>2</sub> FPKM) for medulloblastoma samples grouped by molecular subtype. Adjusted p-values are shown.](https://raw.githubusercontent.com/AlexsLemonade/OpenPBTA-analysis/0107be711b79f228f858f539fad1a4e8b2ced7c3/figures/pngs/figure5.png?sanitize=true){#fig:Fig5 width="7in"}
\ No newline at end of file
+Indeed, we detected significant (Bonferroni-corrected p < 0.05) upregulation of _MYC_ and _E2F_ targets, as well as G2M (cell cycle phase following S phase) in both embryonal and diffuse astrocytic and oligodendroglial tumors compared to several other cancer groups.
+In contrast, low-grade astrocytic tumors showed significant downregulation (Bonferroni-corrected p < 0.05) of these pathways.
+Schwannomas and neurofibromas, which have a documented inflammatory immune microenvironment of T and B lymphocytes as well as tumor-associated macrophages (TAMs), are driven by upregulation of cytokines such as IFN$\gamma$, IL-1, and IL-6, and TNF$\alpha$ [@doi:10.1093/noajnl/vdaa023].
+We observed this trend in GSVA hallmark pathways (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**) and immune cell deconvolution, which showed that monocytes dominated the identified immune cells in these tumors (**Figure {@fig:Fig5}C**).
```

Comment:
```suggestion
Indeed, we observed significant upregulation of these cytokines in GSVA hallmark pathways (Bonferroni-corrected p < 0.05) (**Figure {@fig:Fig5}B**) and found immune cell types dominated by monocytes in these tumors (**Figure {@fig:Fig5}C**).
```

I think of "trend" as synonymous with "not-significant/near-significant", so maybe we can reword